guides/doc-Monitoring_Project/topics/con_metrics-data-retention.adoc

```diff
@@ -0,0 +1,13 @@
+[id='metrics-data-retention_{context}']
+= Metrics Data Retention
+
+The storage capacity required by PCP data logging is determined by the following factors:
```

Comment:
Is it possible / reasonable to provide a basic example? e.g. For 60 second interval, 14 day retention with default metrics it requires approximately X amount of space?

Reply:
on our setup it's been something like 1MB per day (compressed), which roughly matches https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/monitoring_and_managing_system_status_and_performance/setting-up-pcp_monitoring-and-managing-system-status-and-performance#example-analyzing-the-centralized-logging-deployment_setting-up-pcp which says 10 hosts at 60 sec sampling does 20MB

Reply:
and the uncompressed requirements are documented in "Confirming Data Storage Usage" :)